ajax/libs/vonic/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "vonic",
+  "version": "1.1.2",
+  "filename": "vonic.min.js",
+  "description": "Mobile UI Components, based on Vue.js and ionic CSS.",
+  "keywords": [
+    "vonic",
+    "vonic framework",
+    "vue",
+    "ionic",
+    "app",
+    "mobile",
+    "components",
+    "ui components",
+    "spa"
+  ],
+  "author": "wangdahoo <157195705@qq.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wangdahoo/vonic/"
+  },
+  "license": "MIT",
+  "homepage": "https://wangdahoo.github.io/vonic-documents",
```

Comment:
I think docs in not homepage. The one you provide in the PR description is better.
`https://wangdahoo.github.io/vonic/docs/`
What do you think?